ios/brave-ios/Sources/Brave/Frontend/Shields/SubmitReportView.swift

```diff
@@ -127,7 +175,10 @@ struct SubmitReportView: View {
     .navigationViewStyle(.stack)
   }
 
-  @MainActor func createAndSubmitReport() async {
+  @MainActor func createAndSubmitReport() async -> Bool {
+    guard !isSubmitDisabled else {
```

Comment:
why do you need this to return a Bool if we are updating a @State isSubmitted?

Reply:
Yeah good point, will remove and use `isSubmitted` instead!